packages/@aws-cdk/aws-kinesisfirehose-destinations/lib/s3-bucket.ts

```diff
@@ -2,13 +2,25 @@ import * as iam from '@aws-cdk/aws-iam';
 import * as firehose from '@aws-cdk/aws-kinesisfirehose';
 import * as s3 from '@aws-cdk/aws-s3';
 import { Construct } from 'constructs';
-import { CommonDestinationProps } from './common';
+import { CommonDestinationProps, Compression } from './common';
 import { createLoggingOptions } from './private/helpers';
 
 /**
  * Props for defining an S3 destination of a Kinesis Data Firehose delivery stream.
  */
-export interface S3BucketProps extends CommonDestinationProps { }
+export interface S3BucketProps extends CommonDestinationProps {
+  /**
+   * The type of compression that Kinesis Data Firehose uses to compress the data
+   * that it delivers to the Amazon S3 bucket.
+   *
+   * The compression formats SNAPPY or ZIP cannot be specified for Amazon Redshift
+   * destinations because they are not supported by the Amazon Redshift COPY operation
+   * that reads from the S3 bucket.
```

Comment:
I fee like this comment is not needed, since this is the `S3Destination`...?

Reply:
Good point, this comment will be needed once this prop is inevitably hoisted up to another interface so it can be shared with multiple destinations. But we can delete for now.